.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,13 +2,13 @@
 
 ------------------------
 
+Hello! Thanks for your contribution! Please mention someone in a comment (`@username`) to speed up the reviewing process.  See [Project Committee](https://projects.apache.org/committee.html?beam) page, or simply tag a suggested reviewer if you don't have anyone on your mind. 
+
```

Comment:
I would replace 

> or simply tag a suggested reviewer if you don't have anyone on your mind

>  or simply tag **one of the** suggested reviewers **in the github list** if you don't have anyone on your mind

for it to be even more clear

Reply:
And maybe but this sentence in bold

Reply:
I think the point of this blurb is not (necessarily) to tell an author how to find a reviewer, but to make them aware that finding a reviewer is a good next step. 

My suggestion would be something along the lines of 

"After creating the pull request please, please [choose a reviewer](relevant contrib guide paragraph) and mention them in a comment (`@username`) to speed up the reviewing process, or simply tag **one of the** suggested reviewers **in the github list** if you don't have anyone on your mind. Mapping from the project committee list to a github id may be non-obvious. 

Should this be part of the checklist? 

(I don't want to bikeshed on this, anything is better than nothing, but that's my two cents.)

Reply:
> I think the point of this blurb is not (necessarily) to tell an author how to find a reviewer, but to make them aware that finding a reviewer is a good next step.

Fair point. Given that Beam Contribution Guite will be linked and shows the ways of finding the reviewer I think it's enough. 

> Should this be part of the checklist?

It should be, I guess. But it should be on the top of it to draw atention the most.